cmd/validator/flags/flags.go

```diff
@@ -351,6 +351,13 @@ var (
 		Usage: "Enables Builder validator registration APIs for the validator client to update settings such as fee recipient and gas limit. Note* this flag is not required if using proposer settings config file",
 		Value: false,
 	}
+
+	// BuilderGasLimitFlag defines the gas limit for the builder to use for constructing a payload.
+	BuilderGasLimitFlag = &cli.IntFlag{
+		Name:  "gas-limit",
```

Comment:
should we call this suggested-gas-limit? it doesn't set the gas limit it sets the gas limit to strive towards.

Reply:
sure!